### DISCUSSION TITLE
Is Dask distributing files to nodes?

### DISCUSSION THREAD

**Discussion by odo2063** [dask/dask#9618] [Q&A] [answered]
Hi Everybeeing!

Could it be that Dask distributes datafiles to the nodes?

imagine something like:
`client.submit(doSomething, someFilename)`
with
`def doSomething(someFilename):
    import json
    data = json.load(someFilename)`

Since I run a cluster file system i see low transferrates but the machine with daskscheduler sends things over the network with it's max rate. If dask distributes these data files how can I switch it off?

**Comment by mrocklin** [comment, marked as answer]
Dask serializes arguments using pickle (or variants thereof).  If you send
a string like "myfilename.txt' then it's just serializing that string and
sending it.  Dask does not know if a string that you provide as an argument
is a filename or not.  It makes no special consideration for strings that
look like filenames.

**Comment by odo2063** [comment]
Thank You very much...It was a configuration issue on my side with the clusterFS, now everything works as expected.
